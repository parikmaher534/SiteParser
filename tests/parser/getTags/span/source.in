<span class="test-span__one">Text</span>

<span class="test-span__two">
	<span class="test-span__inside">Text Inside</span>
</span>
<ul>
	<li><span class="inside-li">SPAN IN LI</span></li>
	<li><span class="inside-li">SPAN IN LI</span></li>
	<li><span class="inside-li">SPAN IN LI</span></li>
	<li><span class="inside-li">SPAN IN LI</span></li>
</ul>
