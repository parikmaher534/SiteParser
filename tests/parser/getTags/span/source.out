<span class="test-span__one">
